### ISSUE TITLE
split term: Multiple Sclerosis

### ISSUE THREAD

**Issue by TudorOprea** [monarch-initiative/mondo#9853]
**Multiple Sclerosis**

sNfL = serum neurofilament light chain.

**1. Early-sNfL Multiple Sclerosis** 
 characterized by sNfL elevation occurring **early** in the disease trajectory

_Sequence of abnormality accumulation (earliest to latest):_
1. Elevated sNfL levels (reflecting active neuronal injury)
2. Reduced T1/T2 ratio in the corpus callosum white matter (indicating microstructural damage)
3. White matter lesion accumulation
4. Volume loss in deep grey matter, limbic cortex, and parietal cortex (occurring later)

_Clinical profile:_
1. Younger patients on average
2. More active inflammatory disease (more gadolinium-enhancing lesions)
3. Faster brain atrophy rates
4. Higher risk of developing new lesions (144% increased hazard compared to the other subtype in one cohort)
5. More responsive to treatment in terms of reducing active lesion counts
6. Faster decline in sNfL levels over time, particularly with treatment 

**2. Late-sNfL Multiple Sclerosis** 
 characterized by sNfL elevation occurring **later** in the disease trajectory

_Sequence of abnormality accumulation (earliest to latest):_
1. Volume loss in limbic cortex (early atrophy without elevated blood markers)
2. Deep grey matter volume loss
3. White matter lesion accumulation and corpus callosum microstructural damage
4. Elevated sNfL levels occurring at a later stage

_Clinical profile:_
1. Older patients, higher proportion of women
2. More disabled at baseline
3. Slower brain atrophy rates
4. Lower risk of developing new lesions
5. Less responsive to treatment in terms of active lesion reduction
7. More stable sNfL levels over time

** Biomarkers and MRI Measures Used**
1. Serum neurofilament light chain  - a marker for neuroaxonal damage released during neuronal injury
2. MRI - limbic cortex volume
3. MRI - deep grey matter volume
4. MRI - parietal cortex volume
5. MRI - total lesion volume
6. MRI - corpus callosum white matter T1/T2 ratio

**Source**

[MultipleSclerosis_Subtypes_Brain2025_awaf331.pdf](https://github.com/user-attachments/files/24388379/MultipleSclerosis_Subtypes_Brain2025_awaf331.pdf)

**ORCID**
https://orcid.org/0000-0002-6195-6976

**Comment by sabrinatoro** [comment]
# Issue analysis

> **WARNING: This report is AI-generated**
> **Model:** claude-opus-4-6
> **Generated:** 2026-03-30 16:59:24 UTC

## Request Summary

The issue requests splitting Multiple Sclerosis (MONDO:0005301) into two new subtypes:
1. **Early-sNfL Multiple Sclerosis** - characterized by sNfL elevation occurring early in the disease trajectory
2. **Late-sNfL Multiple Sclerosis** - characterized by sNfL elevation occurring later in the disease trajectory

Based on: Willard et al. (2025) "Combined magnetic resonance imaging and serum analysis reveals distinct multiple sclerosis types." *Brain* 148:4578-4591. doi:10.1093/brain/awaf331

## Current MONDO Hierarchy for Multiple Sclerosis

```mermaid
graph BT
    RRMS[MONDO:0005314<br/>relapsing-remitting MS] --> MS[MONDO:0005301<br/>multiple sclerosis]
    CPMS[MONDO:0005284<br/>chronic progressive MS] --> MS
    SPMS[MONDO:0000450<br/>secondary progressive MS] --> CPMS
    PPMS[MONDO:0000451<br/>primary progressive MS] --> CPMS
    PRMS[MONDO:0000452<br/>progressive relapsing MS] --> CPMS
    MAMS[MONDO:0016429<br/>Marburg acute MS] --> MS
    PEMS[MONDO:0018784<br/>pediatric MS] --> MS
```

## Validity Assessment: Concerns with this Request

### 1. Data-derived subtypes, not established disease entities

The proposed subtypes were identified through **unsupervised machine learning** (SuStaIn model) applied to MRI + sNfL biomarker data. They are not clinically recognized disease entities with established diagnostic criteria. The paper itself describes these as "data-derived subtypes" and "biologically informed MS subtypes" - they are research constructs, not disease classifications adopted by clinical consensus bodies.

### 2. Single study, not independently replicated

This classification comes from a **single research group** using a specific computational pipeline. While the study validated across two clinical trial cohorts (training n=189, external test n=445), this is internal validation within a single study. The subtypes have not been independently replicated by other groups or adopted into any clinical classification system (e.g., McDonald criteria, which define current MS subtypes).

### 3. Subtypes represent a continuum, not discrete entities

The paper explicitly states: *"these subtypes are likely to represent a continuum of underlying pathology"* (p. 4588). Approximately **7% of high-confidence patients in the training set and 23% in the test set switched subtypes** over time. This instability is inconsistent with discrete disease entities suitable for ontology representation.

### 4. No established diagnostic criteria

Classifying a patient into these subtypes requires:
- Specific MRI processing pipeline (SuStaIn framework)
- sNfL laboratory measurements
- Application of a trained machine learning model

There are no clinical diagnostic criteria that a physician could apply. The existing MS subtypes in MONDO (RRMS, SPMS, PPMS, PRMS) are based on **decades of clinical observation** and **international consensus criteria** (Lublin et al., 1996; 2014).

### 5. Orthogonal classification axis

These biomarker-derived subtypes cut **across** established clinical phenotypes. The paper shows both RRMS and SPMS patients distributed across both early-sNfL and late-sNfL groups. They represent a different axis of classification (biomarker trajectory) rather than distinct diseases. In ontological terms, these are better modeled as **disease stages or biomarker profiles** rather than disease subtypes.

### 6. Comparison with existing MONDO subtypes

| Feature | Existing MS subtypes (RRMS, SPMS, etc.) | Proposed sNfL subtypes |
|---|---|---|
| Basis | Clinical course observation | ML clustering of biomarkers |
| Diagnostic criteria | International consensus (McDonald) | Requires specific ML model |
| Clinical use | Standard of care | Research only |
| Stability | Defined clinical transitions (e.g., RRMS→SPMS) | 7-23% switch between subtypes |
| Independent validation | Decades of clinical use | Single study, two cohorts |
| Recognition | WHO (ICD), SNOMED, OMIM, etc. | None |

## Recommendation

**This request should be declined at this time.** The proposed subtypes do not meet the criteria for inclusion in MONDO as disease entities because:

- They are research-stage biomarker clusters from a single machine learning study
- They lack independent replication and clinical consensus
- They represent a continuum rather than discrete disease entities
- No clinical diagnostic criteria exist
- They are orthogonal to (not replacements for) the existing clinically-established MS classification

### What would change this assessment?

The subtypes could be reconsidered for MONDO inclusion if:
1. **Independent replication** by multiple research groups using independent cohorts
2. **Clinical consensus** adoption (e.g., incorporation into revised McDonald criteria or similar clinical guidelines)
3. **Established diagnostic criteria** that can be applied without the specific ML pipeline
4. **Recognition by major classification systems** (ICD, OMIM, Orphanet, etc.)

### Alternative suggestions

If the requester's goal is to capture biomarker-based MS stratification:
- Consider whether this is better represented as an **annotation or property** on the existing MS term (e.g., a note about emerging biomarker-based subtyping research)
- The paper could be added as a **seeAlso** reference on MONDO:0005301 to track emerging research
- This classification may be more appropriate for a **research-focused resource** rather than a disease ontology

**Comment by sabrinatoro** [comment]
Thank you for this request, @TudorOprea 
I reviewed the publication you shared (Willard et al, 2025) and your request. _(Note that the analysis in the comment above confirmed my conclusions)_

I do not think that "Early-sNfL Multiple Sclerosis" and "Late-sNfL Multiple Sclerosis" terms are appropriate in Mondo (at least at this point). These subtypes have been identified via a machine learning model, and have not been /are not used as subtypes in clinics or diagnosis. 

I think this information would be more relevant in a resource where disease and phenotype progressions are curated, but not in the ontology. 

I am happy to discuss further if you disagree and have arguments for these subtypes to be created as new terms. 
Thank you.
